OpenWrt master requires at least CMake 3.5

Compiling uspot on the current master branch of OpenWrt fails with the following error:

```
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
```

This happens because recent versions of CMake require at least version 3.5, this file needs to be updated: https://github.com/f00b4r0/uspot/blob/next/src/CMakeLists.txt#L1.

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.5)
 
 PROJECT(uspot C)
 INCLUDE(GNUInstallDirs)

